src/openassetio-python/tests/package/hostApi/test_manager.py

```diff
@@ -567,6 +567,180 @@ def test_when_relationship_is_None_then_exception_raised(
         assert not mock_manager_interface.mock.getWithRelationships.called
 
 
+class Test_Manager_getWithRelationshipPaged:
+    def test_wraps_the_corresponding_method_of_the_held_interface(
+        self,
+        manager,
+        a_ref,
+        mock_manager_interface,
+        mock_entity_reference_pager_interface,
+        a_host_session,
+        a_batch_element_error,
+        an_empty_traitsdata,
+        an_entity_trait_set,
+        an_entity_reference_pager,
+        a_context,
+        invoke_getWithRelationshipPaged_success_cb,
+        invoke_getWithRelationshipPaged_error_cb,
+    ):
+        # pylint: disable=too-many-locals
+
+        two_refs = [a_ref, a_ref]
+        page_size = 3
+
+        success_callback = mock.Mock()
+        error_callback = mock.Mock()
+
+        method = mock_manager_interface.mock.getWithRelationshipPaged
+
+        def call_callbacks(*_args):
+            invoke_getWithRelationshipPaged_success_cb(0, mock_entity_reference_pager_interface)
+            invoke_getWithRelationshipPaged_error_cb(1, a_batch_element_error)
+
+        method.side_effect = call_callbacks
+
+        manager.getWithRelationshipPaged(
+            two_refs,
+            an_empty_traitsdata,
+            page_size,
+            a_context,
+            success_callback,
+            error_callback,
+            resultTraitSet=an_entity_trait_set,
+        )
+
+        method.assert_called_once_with(
+            two_refs,
+            an_empty_traitsdata,
+            an_entity_trait_set,
+            page_size,
+            a_context,
+            a_host_session,
+            mock.ANY,
+            mock.ANY,
+        )
+        success_callback.assert_called_once_with(0, mock.ANY)
+
+        # Additional assert to check cpp constructed Pager object is as
+        # expected.
+        pager = success_callback.call_args[0][1]
```

Comment:
[fix] Can we also test with `ininstance(pager, EntityReferencePager)` to make sure the pager is definitely wrapper?